plugins/modules/elb_target_group.py

```diff
@@ -89,6 +89,12 @@
     required: false
     choices: [ 'http', 'https', 'tcp', 'tls', 'udp', 'tcp_udp', 'HTTP', 'HTTPS', 'TCP', 'TLS', 'UDP', 'TCP_UDP']
     type: str
+  protocol_version:
+    description:
+      - Specifies protocol version.
+    required: false
+    choices: ['GRPC', 'HTTP1', 'HTTP2']
+    type: str
```

Comment:
```suggestion
  protocol_version:
    description:
      - Specifies protocol version.
    required: false
    choices: ['GRPC', 'HTTP1', 'HTTP2']
    type: str
    version_added: 4.3.0
```

But I'm not sure if we are fast enough here ...maybe it becomes 5.1.0

Reply:
Hi @markuman, thank you for the suggestion. I've added `4.3.0` for now, if `4.3.0` gets released before this PR is merged, I'll modify it to `5.1.0`.

Reply:
Yes, it must be now `5.1.0` @sharvarikhedkar

Reply:
Thank you @markuman, updated it to `5.1.0`